awaitility-groovy/pom.xml

```diff
@@ -54,6 +54,15 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
What's the reason for adding these settings to the surefire plugin? Are they required for end users as well!?